Sources/Swift/Protocol/Codable/SentryLogCodable.swift

```diff
@@ -0,0 +1,46 @@
+@_implementationOnly import _SentryPrivate
+import Foundation
+
+extension SentryLog: Codable {
+    
+    private enum CodingKeys: String, CodingKey {
+        case timestamp
+        case traceId = "trace_id"
+        case level
+        case body
+        case attributes
+        case severityNumber = "severity_number"
+    }
+    
+    public func encode(to encoder: any Encoder) throws {
+        var container = encoder.container(keyedBy: CodingKeys.self)
+        
+        try container.encode(sentry_toIso8601String(timestamp), forKey: .timestamp)
```

Comment:
Not sure if using `sentry_toIso8601String` within the encode function is the way to go. Probably better to set it as a encoding strategy?

Reply:
In this case I would also consider using the date strategy of the encoder.

cc @philipphofmann as you implemented Encodable for multiple types in the SDK a while ago

Reply:
Introduced an encoding helper function, just like the one we have for decoding.